src/test/java/de/fraunhofer/aisec/cpg/enhancements/variable_resolution/VariableResolverCppTest.java

```diff
@@ -341,7 +368,7 @@ void testAccessStaticMemberThroughInstanceFirst() {
     VRUtil.assertUsageOfMemberAndBase(
         callParamMap.get("func5_staticVarName_throughInstance_first"),
         declaration,
-        outerStaticVarName); // external_static_field
+        outerStaticVarName.getDefinition()); // external_static_field
```

Comment:
Ah! external static fields might be a use case for declaration vs. definition of fields, but not every un-initialized field is undefined

Reply:
Yes, with regards to the above comments, there can be a definition of a field and a declaration outside of the class, while in most cases I guess that declarations without initializer can have a default value, there are these static field declarations where there is an actual declaration that we need to know about